tests/conftest.py

```diff
@@ -28,3 +28,14 @@ def sample_service():
 def rmock():
     with requests_mock.mock() as rmock:
         yield rmock
+
+
+@pytest.fixture
+def app_with_statsd(app):
+    app.config['NOTIFY_ENVIRONMENT'] = "test"
+    app.config['NOTIFY_APP_NAME'] = "api"
```

Comment:
```suggestion
    app.config['NOTIFY_APP_NAME'] = "utils"
```
Not really important, feel free to disregard